std/algorithm/searching.d

```diff
@@ -2555,6 +2555,18 @@ template canFind(alias pred="a == b")
     ];
     assert(!canFind(words, "bees"));
     assert( canFind!((string a, string b) => a.startsWith(b))(words, "bees"));
+
+}
+
+/// Search for mutliple items in an array of items (searching needles in a hay stack)
```

Comment:
Typo: multiple
Also the description is wrong. You are searching for items in an array of array of items. Though that will be confusing to people.
Maybe something like:

Search for needles in an array of hay stacks

Reply:
@wilzbach I force pushed. Does the new changeset look OK?